proto/pocket/migration/tx.proto

```diff
@@ -94,7 +100,7 @@ message MsgClaimMorseAccount {
   // This account is liable for any fees incurred by violating the constraints of Morse
   // account/actor claim message fee waiving; the tx contains ONE OR MORE Morse account/actor
   // claim messages AND has EXACTLY ONE signer.
-  string shannon_signing_address = 4 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "shannon_signing_address"];
+  string shannon_signing_address = 5 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "shannon_signing_address"];
```

Comment:
We should not be changing the index of an existing type/field.

Reply:
I think there may have been some reversion during a merge conflict resolution. :thinking: